unlock-protocol.com/package.json

```diff
@@ -46,10 +46,6 @@
     "@svgr/cli": "4.3.2",
     "nodemon": "1.19.2"
   },
-  "engines": {
-    "node": "8.16.1"
```

Comment:
I think we need to keep a hardcoded version in there so we have some "locked" version to make sure things run well!

Reply:
hmm, can we lock it to the current LTS version then?  10.16.3